src/helpers/isHotPost.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  minimumLikes,
+  minimumNumberOfComments,
+  minimumParticipants,
+  minimumViews,
+  relevancePeriod,
+} from '@/data/hotPost'
+import getFeedsContract from '@/helpers/getFeedsContract'
+import getViews from '@/helpers/postViews'
+
+async function getParticipants(
+  feedId: number,
+  postId: number,
+  skip = 1,
+  pinned = false
+) {
+  const [{ participants }] = await getFeedsContract.getPostsAndParticipants(
+    feedId,
+    postId,
+    skip,
+    pinned
+  )
+
+  return participants.length
+}
+
+async function getNumberOfComments(feedId: number, postId: number) {
+  const lastCommentId = await getFeedsContract.lastCommentIds(feedId, postId)
+  return lastCommentId.toNumber()
+}
+
+async function getNumberOfLikes(feedId: number, postId: number) {
+  const [, likes] = await getFeedsContract.getReactions(feedId, postId, 0)
+  return likes.toNumber()
+}
+
+async function isOldPost(feedId: number, postId: number) {
+  const post = await getFeedsContract.posts(feedId, postId)
+  return Date.now() - post.timestamp.toNumber() * 1000 > relevancePeriod
+}
+
+export default async function isHotPost(feedId: number, postId: number) {
+  const isOld = await isOldPost(feedId, postId)
+  if (isOld) return false
+
+  const participants = await getParticipants(feedId, postId)
+
+  if (participants < minimumParticipants) return false
```

Comment:
Should we exit that soon? Post has 20 upVotes and 2 commentators, we won't send `hotPost` notification